requirements.txt

```diff
@@ -4,3 +4,7 @@ blosc>=1.4.4
 six
 mock
 nose2
+
+# cloud-volume
```

Comment:
Can remove this.

Reply:
sorry, my bad — this was to document that the below are for cv installs only. (we'll eventujally want to separate this out as `pip install intern[cloudvolume]` or something, so i figured i'd keep them documented.

Reply:
Oh gotcha. Didn't realize that this indicates the stuff below if cv specific. Thought it was a repeat of the cv requirement.